### PR TITLE
Fix math equation rendering format in api definitions

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix mess-up math equation format in api definition related pbtxts according to the [Math in markdown guideline](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/community/documentation.md#math-in-markdown).

Take [tf.gather_nd](https://www.tensorflow.org/api_docs/python/tf/gather_nd) as an example below:
Before:
image
After:
image

### CODE REVIEW

**Comment by jhseu** [comment]
Adding Martin because if we decide to change the formatting here, it might be better to change everything at once.

**Comment by martinwicke** [comment]
@MarkDaoust will this render correctly? Some of these are MathJax inside code blocks. I'm happy with this change if it works.

File: tensorflow/core/api_def/base_api/api_def_Exp.pbtxt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 op {
   graph_op_name: "Exp"
-  summary: "Computes exponential of x element-wise.  \\\\(y = e^x\\\\)."
+  summary: "Computes exponential of x element-wise.  \\(y = e^x\\)."
```
Comment (MarkDaoust) [inline tensorflow/core/api_def/base_api/api_def_Exp.pbtxt]: These ones, in .pbtxt files, need to stay double-escaped.

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
hmmm.... 

https://www.tensorflow.org/api_docs/python/tf/zeta

It looks like a lot of the spots where you reduced the escaping one level on the `\\(` are already rendering correctly. So I think the reduced escaping will break them. you need to use `\\\\(` in the `.pbtxt` files.

They're not rendering on the cc pages:

https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/zeta

It looks like the cc pages just don't have mathjax installed, but that's a separate problem.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrewharp: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by MarkDaoust** [comment]
>It looks like a lot of the spots where you reduced the escaping one level on the \\( are already rendering correctly. So I think the reduced escaping will break them. you need to use \\\\( in the .pbtxt files.

Nope. I was wrong about this.

It's the summary line that looses one level of escaping. The summary lines (like the change to Exp) need the double escaping \\\\( \\\\). The description fields need the single escaping (that's why [polygamma](https://www.tensorflow.org/api_docs/python/tf/polygamma) already works).

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**